src/js/plugin/sdk/services/MarathonService.js

```diff
@@ -0,0 +1,22 @@
+
+import config from "../config/config";
+import ajaxWrapper from "../../../helpers/ajaxWrapper";
+
+export default class MarathonService {
+
+  request(opts = {}) {
+    let method = "GET";
+
+    if (opts.method) {
+      method = opts.method;
+    }
+
+    return ajaxWrapper({
```

Comment:
I wonder if it makes sense to always allow additional option to pass `embed` flags.

Reply:
Currently it is possible to pass all needed embeds directly on the `resource` attribute, like this:

```javascript
MarathonService.request({
  resource: "/v2/groups?embeds=...&embeds=..."
});
```

Do you think we need a new attibute specific for this on the object passed to `request()`?

Maybe this makes more sense on `MarathonActions.getGroup()`, since there we have a higher level API.

What do you think?

Reply:
Fair point. Let's add these flags to the higher level API.